src/main/java/com/hazelcast/web/WebFilterConfig.java

```diff
@@ -37,30 +37,135 @@
  * Contains all configuration parameters for Hazelcast session replication
  */
 public final class WebFilterConfig {
-    private static final ILogger LOGGER = Logger.getLogger(WebFilterConfig.class);
+    /**
+     * <p>Configuration key: <code>client-config-location</code></p>
```

Comment:
Can you remove all these `Configuration key` sections? I think they're not necessary.

Reply:
Sure